osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/application/config/Iec61850Config.java

```diff
@@ -96,48 +90,54 @@ public int iec61850RtuPortServer() {
         return Integer.parseInt(this.environment.getRequiredProperty(PROPERTY_NAME_IEC61850_RTU_PORT_SERVER));
     }
 
+    @Bean(name = "protocolAdapterIec61850NettyServerBossGroup")
+    public DisposableNioEventLoopGroup serverBossGroup() {
+        // TODO - Should we pass in any parameters, like for example number of
+        // threads, to the constructor?
+        return new DisposableNioEventLoopGroup();
+    }
+
+    @Bean(name = "protocolAdapterIec61850NettyServerWorkerGroup")
+    public DisposableNioEventLoopGroup serverWorkerGroup() {
+        // TODO - Should we pass in any parameters, like for example number of
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)